open_lm/train.py

```diff
@@ -133,7 +133,7 @@ def train_one_epoch(model, data, loss, epoch, optimizer, scaler, scheduler, args
         if args.accum_freq == 1:
             with autocast():
                 inputs, targets = sample_chunk(texts, args.seq_len, args.target_mask_left)
-                out, _ = model(inputs)
+                out, _, _ = model(inputs)
```

Comment:
We might want to change our model output to be a class so we don't have to make changes like these too often. But for now this is fine.

Reply:
Actually we could align that class with the one already used by HF. I think it would be better, should it be part of this PR?

Reply:
yeah agreed, but let's leave it for a later PR